### PR TITLE
Camera Module Tutorial

### PR DESCRIPTION
**Pull request by Frijol**
Adds tutorial for ~~video~~ camera module.

Still needs:
- [x] Any special instructions?
- [x] Example
- [x] GIF of example's results
- [x] Description of results for Step 5
- [x] Bonus problem

@rwaldron I built out the scaffolding so that when you're ready it will be easy to add in.

### CODE REVIEW

**Comment by rwaldron** [comment]
I'll fill in the blanks on this

**Comment by rwaldron** [comment]
@Frijol updated!

File: modules/camera.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+### Step 2
+</div>
+</div>
+
+<div class="row">
+<div class="large-6 columns">
+
+Plug Tessel into your computer via USB, then plug the camera module into either of Tessel's USB ports. You will also need a UVC compatible USB camera. For this tutorial, either of these is suitable: 
+
```
Comment (Frijol) [inline modules/camera.md]: It's a bit weird to list these here- why not refer back to tessel.io/modules?

File: modules/camera.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+var http = require('http');
+var port = 8080;
+var camera = new av.Camera();
+
+http.createServer((request, response) => {
+  response.writeHead(200, { 'Content-Type': 'image/jpg' });
+
+  camera.capture().pipe(response);
+
+}).listen(port, () => console.log(`http://${os.hostname()}.local:${port}`));
```
Comment (Frijol) [inline modules/camera.md:63]: looks like the wrong quote types

File: modules/camera.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+</div>
+
+<div class="row">
+<div class="large-6 columns">
+
+In your command line, `t2 run camera.js`
+
+Hooray! You should see the following in your terminal: 
+
+```
```
Comment (Frijol) [inline modules/camera.md]: This display type doesn't process properly in our t2-start rendering

File: modules/camera.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+$ t2 run camera.js
+INFO Looking for your Tessel...
+INFO Connected to bishop.
+INFO Building project.
+INFO Writing project to RAM on bishop (314.88 kB)...
+INFO Deployed.
+INFO Running camera.js...
+http://bishop.local:8080
+```
```
Comment (Frijol) [inline modules/camera.md]: Why not use our standard of GIF to show output and photo?

File: modules/camera.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+> Note: `bishop` is just the name of my Tessel 2 board, yours will display whatever you've named your board!
+
+**Bonus:** EXTRA CHALLENGE FOR VIDEO
```
Comment (Frijol) [inline modules/camera.md:92]: Do you have a plan for this?

File: modules/camera.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+</div>
+
+<div class="row">
+<div class="large-12 columns">
+
+### Step 6
+
+What else can you do with a camera module? Get inspired by a [community-created project.](http://tessel.io/projects)
+
+- Try connecting a button to your Tessel 2 and use it as a shutter.
```
Comment (Frijol) [inline modules/camera.md]: Oh - can this go in the Bonus section above?

File: modules/camera.md (hunk 1 above)
Comment (rwaldron) [inline modules/camera.md]: :ballot_box_with_check:

File: modules/camera.md (hunk 2 above)
Comment (rwaldron) [inline modules/camera.md:63]: Thats a template string.

File: modules/camera.md (hunk 3 above)
Comment (rwaldron) [inline modules/camera.md]: Whoops... I'll fix those in the other PR as well.

File: modules/camera.md (hunk 6 above)
Comment (rwaldron) [inline modules/camera.md]: :ballot_box_with_check:

File: modules/camera.md (hunk 4 above)
Comment (rwaldron) [inline modules/camera.md]: :ballot_box_with_check:

**Comment by johnnyman727** [comment]
@rwaldron can we merge this and deploy?

**Pull request merged**